src/Plivo/Client/SystemHttpClient.cs

```diff
@@ -112,7 +112,7 @@ public SystemHttpClient(BasicAuth basicAuth, Dictionary<string, string> proxySer
                 );
             _client.DefaultRequestHeaders.Authorization = authHeader;
             _client.DefaultRequestHeaders.Add("User-Agent", "plivo-dotnet/" + ThisAssembly.AssemblyVersion);
-            var baseServerUri = string.IsNullOrEmpty(baseUri) ? "https://api.plivo.com/" + Version.ApiVersion  : baseUri;
+            var baseServerUri = string.IsNullOrEmpty(baseUri) ? "https://api.numbers.plivodev.com/" + Version.ApiVersion  : baseUri;
             _client.BaseAddress = new Uri(baseServerUri + "/");
```

Comment:
Todo @nikhil-plivo Revert back post tetsing.

Reply:
I suggest better you revert this now. Let the QA take care of this.